tests/test_defects.py

```diff
@@ -151,6 +151,40 @@ def check_maskBlocks(self, inputDefects, expectedDefects):
         for boxMeas in boxesMeasured:
             self.assertIn(boxMeas, unionInputExpectedBoxes)
 
+    def test_normalize_defects(self):
+        """A test for the lsst.meas.algorithms.Defect.normalize() method.
```

Comment:
Why does this live in `cp_pipe`? Shouldn't the tests for `lsst.meas.algorithms.Defect` live in `lsst.meas.algorithms`?

Reply:
I'll move it to somewhere in "meas_algorithms/tests/".